osu.Game/OsuGame.cs

```diff
@@ -220,6 +220,11 @@ public partial class OsuGame : OsuGameBase, IKeyBindingHandler<GlobalAction>, IL
 
         private readonly List<OverlayContainer> visibleBlockingOverlays = new List<OverlayContainer>();
 
+        /// <summary>
+        /// Whether the game should be limited to only display licensed content.
+        /// </summary>
+        public bool HideUnlicensedContent => RuntimeInfo.OS == RuntimeInfo.Platform.iOS;
```

Comment:
Will this also happen on Android? There was no context why it's only limited to iOS.

And this doesn't prevent users from just downloading `.osz`s from safari/browser and importing, copying an existing desktop installation to macOS Finder or iTunes, and clicking non-featured artist beatmap links on chat/multiplayer/playlists.

Unless there is some stringent checks coming later (i.e. logging in before importing and checking if the beatmap is from a featured artist and limiting `BeatmapSetOverlay` links), users have these workarounds.

Reply:
Intentional.